.github/workflows/wolfi-presubmit.yaml

```diff
@@ -154,6 +156,17 @@ jobs:
         run: |
           make SHELL="/bin/bash" MELANGE="sudo melange" test/${{ matrix.package }}
 
+      - name: Run tests to verify xattrs with QEMU runner
+        if: matrix.runner == 'qemu' && matrix.package == 'fping'
```

Comment:
I was thinking that we should probably always do this.  We can exclude stuff that fails (with a TODO to fix them)?

Reply:
Ok moving to run all tests with qemu